cloud-service-providers/azure/aks/setup/README.md

```diff
@@ -0,0 +1,48 @@
+# Setup AKS
+
+The keys to create AKS for NIM is to create proper GPU nodepool.  Details are in the following table. 
+
+## Connect to Azure
+
+```
+az login --use-device-code
+az account set --subscription <subscription name>
+```
+
+## Create AKS
+
+```
+az aks create -g <resource group name> -n <aks name> --location <location has desired GPU> --zones <zone has desired GPU> --generate-ssh-keys
+```
+
+## Create GPU nodepool
+
+```
+az aks nodepool add --resource-group <resource group name> --cluster-name <aks name> --name <nodepool name> --node-count 1 --skip-gpu-driver-install --node-vm-size <Desire VM type> --node-osdisk-size 2048 --max-pods 110
+```
+
+|    Desire VM type                                  | Llama-3-8b    | Llama-3-70b   |
```

Comment:
Is this documented somewhere on the official NIM docs? Feels like if we publish this here it will quickly become out of date.

I would rather see us point towards the official support matrix and have this information added there if possible.

Reply:
done